### PR TITLE
Add compat data for CSS motion path properties

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds data for:

* [`offset`](https://developer.mozilla.org/docs/Web/CSS/offset)
* [`offset-distance`](https://developer.mozilla.org/docs/Web/CSS/offset-distance)
* [`offset-path`](https://developer.mozilla.org/docs/Web/CSS/offset-path)
* [`offset-rotate`](https://developer.mozilla.org/docs/Web/CSS/offset-rotate)

Let me know if you want to see any changes. Thanks!

### CODE REVIEW

File: css/properties/offset.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      "offset": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/offset",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "55"
+              },
+              {
+                "alternative_name": "offset",
```
Comment (teoli2003) [inline css/properties/offset.json]: "motion"

File: css/properties/offset.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                "version_added": "46"
+              }
+            ],
+            "chrome": [
+              {
+                "version_added": "55"
+              },
+              {
+                "alternative_name": "offset",
```
Comment (teoli2003) [inline css/properties/offset.json]: "motion"

File: css/properties/offset.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                "version_added": "46"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "55"
+              },
+              {
+                "alternative_name": "offset",
```
Comment (teoli2003) [inline css/properties/offset.json]: "motion"

**Comment by ddbeck** [comment]
Fixed. Thank you!

**Pull request merged**